libs/libzauth/libzauth-c/Makefile

```diff
@@ -1,7 +1,7 @@
 LANG := en_US.UTF-8
 SHELL   := /usr/bin/env bash
 VERSION := "3.0.0"
-ARCH    := amd64
+ARCH    := $(shell if [ -f "`which dpkg-architecture`" ]; then dpkg-architecture -qDEB_HOST_ARCH; else [ -f "`which dpkg`" ] && dpkg --print-architecture; fi )
```

Comment:
That's different from a similar line in the wire-server repo.  Is there a good reason for that?

Reply:
To which other line are you referring? This is the second version of this construct, due to dpkg-dev not being installed on some of our docker images.

Reply:
https://github.com/wireapp/wire-server/blob/7a831025b29d0b9e9d33176a48115f251b578e37/tools/makedeb/Makefile#L5

oops, same line, as it seems.  ignore me!